src/graccreq/oim/OIMTopology.py

```diff
@@ -24,55 +25,60 @@ def __init__(self):
         self.lockfile = lockfile
         self.cachelock = filelock.FileLock(self.lockfile)
 
+        logging.info("Trying to read from cachefile")
         # Lock our cachefile, try to read from it
         with self.cachelock:
-            print "Read lock"
+            logging.debug("Read lock acquired")
             assert self.cachelock.is_locked
             self.have_info = self.read_from_cache()
-        print "Read lock released"
+            logging.debug("Read lock released")
```

Comment:
Is this suppose to be indented?  Different scope than the original print statement.